File: pyStudy/day1227/Study02.java
package day1227;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class Study02 {
    public static void main(String[] args) {
        List<Integer> a = new ArrayList<>(Arrays.asList(1,2,3));
        List<Integer> b = new ArrayList<>(Arrays.asList(4,5,6));    

        List<Integer> combined = new ArrayList<>(a);
        combined.addAll(b);
        System.out.println(combined);

        List<Integer> repeated = new ArrayList<>();
        for(int i=0;i<3;i++){
            repeated.addAll(a);
        }
        System.out.println(repeated);

        System.out.println(a.size());

        System.out.println(a.get(2)+"dog");

        a.set(2,4);
        System.out.println(a);

        a.remove(1);
        System.out.println(a);

        a = new ArrayList<>(Arrays.asList(1,2,3,4,5,6,7,8,9,10));
        a.subList(5, a.size()).clear();
        System.out.println(a);


    }
    


}

File: pyStudy/day1227/study02.py
# 리스트 연산

a = [1,2,3];
b = [4,5,6];

print(a+b);
print(a*3);
print(len(a));

print(str(a[2])+ "dog");

# 리스트의 수정과 삭제
a[2] = 4;
print(a);

# 리스트 요소 삭제
del a[1];
print(a);

a = [1,2,3,4,5,6,7,8,9,10];
del a[5:];
print(a);